### ISSUE TITLE
Alert email syntax problem

### ISSUE THREAD

**Issue by JiffB** [firehol/netdata#3843]
Hi folks,

into file health_alarm_notify.conf, proposed email syntaxes do not work as expected (MUA: claws-mail):
```
EMAIL_SENDER="User Name <user@domain>"      only display: 'User in the From field
EMAIL_SENDER="'User Name' <user@domain>"    only display: 'User in the From field
EMAIL_SENDER="\"User Name\" <user@domain>"  only display: "User in the From field
```
in fact, it only works correctly with a simple address.

**Comment by ktsaou** [comment]
Thank you for reporting this.
Fixed in #3864

**Comment by JiffB** [comment]
You're very welcome.
